src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -1,6 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
```

Comment:
Should we just annotate the whole assembly? We usually don't just annotate single files. @stephentoub what do you think?

Reply:
I didn't really know how to do `TypeConverter` subclasses so left that unannotated

Reply:
I'd prefer such nullable annotations to be done in a dedicated PR and covering more of the assembly in one go.  Nullable annotations require a very specific kind of review, where you review not only what changed, but also what didn't change, since method signatures involving reference types implicitly change meanings when switching into a nullable context.

Reply:
I reverted the nullable annotation changes